config/snapshot.yaml

```diff
@@ -1,13 +1,13 @@
-resolver: lts-12.26
+resolver: lts-13.12
 name: luna
 
 packages:
     - githash-0.1.3.0
+    - containers-0.5.11.0
```

Comment:
I'd appreciate a comment explaining why we need this version pinned down. Later when I review deps list I'm not sure if I should just "bump if it compiles" or if there was some deeper reason to use older version (like changed behavior).